bHY6yh-d42838/cod-bqe7s7_bHY6yh-d42838.xml

```diff
@@ -0,0 +1,407 @@
+               <lb ed="#P"/>autem misericordia iudicium</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e426">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e467">Seneca tragediarum in medea capitulo iiii</ref> 
+               <lb ed="#P"/><quote xml:id="bHY6yh-d42838-Qd1e430">Qui statuit ad parte inaudita altera
+               <lb ed="#P"/>Equum licet statuerit aud equus fuit.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e439">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e481">Idem de tranquillitate animi</ref> <quote xml:id="bHY6yh-d42838-Qd1e443">Semper iudicium
+               <lb ed="#P"/>fauor afficit.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e449">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e492">Idem de clementia</ref> <quote xml:id="bHY6yh-d42838-Qd1e451">prope est vt libenter 
+               <lb ed="#P"/>damnet qui cito prope est ut inimicos 
+               <lb ed="#P"/>puniat / Qui nimis.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e459"> 
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e504">Idem in prouerbium</ref> <quote xml:id="bHY6yh-d42838-Qd1e461">ad penitendum properat qui cito iu<lb ed="#P" break="no"/>dicat.
+               in iudicando criminosa est 
+               <lb ed="#P"/>celeritas.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e469">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e517">Tholomeus in centiloquio capitulo xii</ref> <quote xml:id="bHY6yh-d42838-Qd1e473">beniuo<lb ed="#P" break="no"/>lentia 
+               et maliuolentia homines in 
+               <lb ed="#P"/>iudiciis errare faciunt.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e481">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e531">Lex ff de orig iur l ii</ref> <quote xml:id="bHY6yh-d42838-Qd1e485">per eos qui iuredi<lb ed="#P" break="no"/>ctioni 
+               praesunt iuris effectus accipitur.
+               <lb ed="#P"/>Parum est enim eius ius in ciuitate nisi sint
+               <lb ed="#P"/>qui iura gerere possint.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e495">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d42838-Rd1e547">Lex codi deiudeis l nullus</ref> <quote xml:id="bHY6yh-d42838-Qd1e499">id circho iu<lb ed="#P" break="no"/>diciorum vigor iurisque publici tutela videtur in
+               vigor viris quem publici tutela videtur
+               <lb ed="#P"/>in media constituta ne quisquam sibi permctte<lb ed="#P" break="no"/>re
+               valeat vltionem.</quote>
+            </p>
+            <p xml:id="bHY6yh-d42838-d1e510">
+               <ref xml:id="bHY6yh-d42838-Rd1e563">Ambrosius de officiis libro ii capitulo xxiii</ref> 
+               <quote xml:id="bHY6yh-d42838-Qd1e514">In iudicio gratia absit / causa merita 
+               decernant.</quote>
```

Comment:
what's happening here? Are we missing two line break? If not, then we don't need the hard returns

```suggestion
               <lb ed="#P"/><quote xml:id="bHY6yh-d42838-Qd1e514">In iudicio gratia absit / causa merita 
               <lb ed="#P"/>decernant.</quote>
```